app/services/merchant_service.rb

```diff
@@ -0,0 +1,13 @@
+class MerchantService
+  class << self
+    def conn
+      Faraday.new(url: 'http://localhost:3000/api/v1/merchants')
+    end
+
+    def get_data(url)
+      response = conn.get(url)
+
+      JSON.parse(response.body, symbolize_names: true)
+    end
+  end
+end
```

Comment:
Same thing here.. you're really limited on what you can use the conn method for since you've added the endpoint `api/v1/merchants` onto the domain.